server/gokbg3/grails-app/controllers/org/gokb/UserController.groovy

```diff
@@ -1,9 +1,13 @@
 package org.gokb
 
+import org.springframework.security.access.annotation.Secured
+
+@Secured(['IS_AUTHENTICATED_FULLY'])
 class UserController extends grails.plugin.springsecurity.ui.UserController {
 
   UserProfileService userProfileService
 
+  @Secured(['ROLE_ADMIN', 'IS_AUTHENTICATED_FULLY'])
```

Comment:
no need to secure method agains IS_AUTHENTICATED_FULLY if the whole controller is ...?